tensortrade/env/default/renderers.py

```diff
@@ -559,209 +555,10 @@ def reset(self) -> None:
         self._show_chart = True
 
 
-class MatplotlibTradingChart(BaseRenderer):
```

Comment:
Would you be able to add this class back? And if you can, would you be able to add in those dashboard changes you made?

Reply:
Yep. Will add this class back in. Not too sure how it got lost.